uber/models.py

```diff
@@ -1309,7 +1303,7 @@ def multiply_assigned(self):
 
     @property
     def takes_shifts(self):
```

Comment:
This was broken in the ``c`` refactor and we didn't notice until the unit tests found it out.  ``SHIFTLESS_DEPTS`` used to be a set and now it's a list, so this function now explicitly converts it to a set so we don't need to worry about what type of iterable it is.